### DISCUSSION TITLE
App to always open the latest IDE version

### DISCUSSION THREAD

**Discussion by DinyaZ** [twinbasic/twinbasic#1272] [Ideas]
I have created a simple twinBASIC loader, see it attached in .zip format.
.
Simple app, doesnt need installation, and without UI. It find the newest twinBASIC_IDE_BETA_xxx folder in the "downloads" folder, and run the newest twinbasic.exe.
Place a shortcut icon of the TWBloader to the Windows desktop.
.
It can check if a newer twinBASIC version exist on the GitHub (twinBASIC_IDE_BETA_xxx + 1) and if so, then opens the GitHub releases page on the browser.
.
In setup.ini you can select (true/false):
- delete twinBASIC_IDE_BETA_xxx.zip files
- delete older twinBASIC_IDE_BETA_xxx folders, except for the new, actual one of course. :)
.
Source code in VB6 included in the .zip
.
[twinBasicStarter.zip](https://github.com/twinbasic/twinbasic/files/9795272/twinBasicStarter.zip)

**Comment by dmrvb** [comment]
You are really going to upset the "On Error Resume Next" Police with some of that code :-)

Thank you, its always good to look at somebody else's coding.

**Reply by DinyaZ** [reply]
New version, which can download the new zip, and I hope the OERN-Police will also like it. :)
.
[twinBasicStarter20221029.zip](https://github.com/twinbasic/twinbasic/files/9893206/twinBasicStarter20221029.zip)
